### ISSUE TITLE
printing lr of 1 epoch earlier

### ISSUE THREAD

**Issue by Lee-Gihun** [Lee-Gihun/Pytorch-OSI-setup#28]
scheduler.step is called before learning rate print. This caused the printed learning rate is not current lr, but that of previous step.
